<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1578</width>
    <height>1001</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Chemical Reaction Integrator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1578</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>213</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set reaction equations here!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reaction equation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Actication energy 
 or rate constant</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ylog">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>logarithmic scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushRawdata">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>30</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Export raw data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward reaction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>149</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Backward reaction(optional)</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>181</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="accessibleName">
     <string>Reaction equations</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>412</width>
       <height>434</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="rate_energies">
        <item>
         <widget class="QLineEdit" name="line1_2">
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line2_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line3_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line4_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line5_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line6_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line7_2">
          <property name="maximumSize">
           <size>
            <width>126</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line8_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line9_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line10_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line11_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line12_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line13_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line14_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line15_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="reaction_equations">
        <item>
         <widget class="QLineEdit" name="line1">
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line4"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line5"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line6"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line7"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line8"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line9"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line10"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line11"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line12"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line13"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line14"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line15"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_37"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_38"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_39"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_40"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_41"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_42"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_43"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_44"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_45"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_46"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_47"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_48"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_49"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_50"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_51"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="boolactivation">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>155</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Put activation energies 
 instead of rate constants</string>
    </property>
   </widget>
   <widget class="QChartView" name="chartview">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>851</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPng">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Export png</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_33">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="generate_system">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>172</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Read in System!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>234</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set initial concentrations here!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Species</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>c [mol/L]</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>577</y>
      <width>431</width>
      <height>211</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>412</width>
       <height>434</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_23"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_22"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_21"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_20"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_19"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_18"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_17"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_11"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_10"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_27"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_30"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_29"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_28"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_26"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_25"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_24"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_14"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_16"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_15"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_13"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_12"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_9"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_7"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Starting time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_31">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>600</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>600</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>670</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ending time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_32">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>690</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>690</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>750</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial  time intervall</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_34">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>770</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>770</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>800</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>abs. error</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>800</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rel. error</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>810</y>
      <width>172</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xlog">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>670</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>logarithmic scale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_35">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>820</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_36">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>820</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="safeButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save...</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>232</y>
      <width>72</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="all_energy_options">
     <item>
      <widget class="QCheckBox" name="kcal">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>kcal/mol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ev">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>eV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Hartree">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Hartree</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>generate_system</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>my_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boolactivation</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_energyinput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcal</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_energyunit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ev</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_energyunit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hartree</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_energyunit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calculateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_calculation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>716</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xlog</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_xlog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1221</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>1134</x>
     <y>712</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ylog</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_ylog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>safeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>safeFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>683</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>684</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRawdata</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>export_raw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1351</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>1347</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushPng</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>export_png()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1299</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>1342</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonclicked()</slot>
  <slot>my_slot()</slot>
  <slot>set_energyinput()</slot>
  <slot>set_energyunit()</slot>
  <slot>do_calculation()</slot>
  <slot>set_xlog()</slot>
  <slot>set_ylog()</slot>
  <slot>loadFile()</slot>
  <slot>safeFile()</slot>
  <slot>export_raw()</slot>
  <slot>export_png()</slot>
  <slot>new_window()</slot>
 </slots>
</ui>
